resources/app/snes/metroid3/samus/manifests/layout.json

```diff
@@ -800,50 +800,57 @@
     "stand_left1": {
       "parent": "stand_left",
       "used by": [
-        ["0x02", 1]
+        ["0x02", 1],
+        ["0x8A", 1]
```

Comment:
this change will add a buffer frame to the "running into the wall while facing left" animation sequence.  The effect of this should not be noticeable by casual players, but this does provide additional buffering against bad DMA call.  Seems like a good change.